app/models/message.rb

```diff
@@ -73,52 +60,7 @@ def remove_message_id_in_body
       record.errors.add :base, 'already saved' # Notifier relies on this message (don't change it)
       record.errors.add :base, :taken
     end
-    if record.subject =~ /Out of Office/i
-      record.errors.add :base, 'autoreply' # don't change!
-    end
-  end
-
-  attr_accessor :dont_send
-
-  after_create :enqueue_send
-
-  def enqueue_send
-    return if dont_send
-    MessageSendJob.perform_later(Site.current, id)
-  end
-
-  def send_message
-    if group
-      send_to_group
-    elsif to
-      send_to_person(to)
-    end
-  end
-
-  def send_to_person(person)
-    if person.email.present?
-      email = Notifier.full_message(person, self, id_and_code)
-      email.add_message_id
-      email.message_id = "<#{id_and_code}_#{email.message_id.gsub(/^</, '')}"
-      email.deliver_now
-    end
-  end
-
-  def send_to_group(sent_to=[])
-    return unless group
-    group.people.each do |person|
-      if should_send_group_email_to_person?(person, sent_to)
-        send_to_person(person)
-        sent_to << person.email
-      end
-    end
-  end
-
-  def should_send_group_email_to_person?(person, sent_to)
-    person.email.present? and
-    person.email =~ VALID_EMAIL_ADDRESS and
-    group.get_options_for(person).get_email? and
-    not sent_to.include?(person.email)
+    record.errors.add :base, 'autoreply' if record.subject =~ /Out of Office/i # don't change!
```

Comment:
The comment `don't change` here applies to the string `'autoreply'` -- not to the regex at the end. I think the meaning of the comment is lost with this as a one-liner.